docs/user/otp-deployment.md

```diff
@@ -0,0 +1,87 @@
+# Setting Up OTP Deployment Servers on Amazon Web Services (AWS)
+
+This document describes how to setup a load balancer server and an UI server for deployment using OTP DataTools, versions prior to Pull Request #506.
```

Comment:
For showing the overall deployment architecture, how about a different doc?

Reply:
I actually don't think it's really necessary to distinguish which version of the application this document applies to at all.

As for putting the architecture diagram in a separate doc, that would be fine. It could have the following sections:
- Deploying Feeds to OpenTripPlanner
-- Overview
-- Deployment Methods
--- Architecture (add diagram)
--- ELB Method (description and link to this doc)
--- S3 Only (short description)
--- Build Graph Over Wire (short description)
-- (other sections?)

Reply:
Let's go ahead and add the new docs to this PR. Now that you've got your 💻 we can get you set up with `mkdocs` so you can build the docs locally.

Why don't you go ahead and update the task assigned to me in #507 to remind me to update the OTP docs when I have implemented that fix. That way it keeps these dependencies out of the docs themselves and in GitHub/our code dev process.

Reply:
Sounds good.